pyproject.toml

```
@@ -0,0 +1,39 @@
+[project]
+authors         = [{ name = "Sam Vente", email = "savente93@proton.me" }]
```

Again, author should be SP devs

```suggestion
authors         = [{ name = "Scientific Python Developers" }]
```